### ISSUE TITLE
Fail to deploy Dotnet Core application

### ISSUE THREAD

**Issue by Lempiraholio** [cloudfoundry/dotnet-core-buildpack#135]
What version of Cloud Foundry and CF CLI are you using? (i.e. What is the output of running `cf curl /v2/info && cf version`?

```
{
   "name": "",
   "build": "",
   "support": "http://support.cloudfoundry.com",
   "version": 0,
   "description": "Cloud Foundry GE Open Sandbox",
   "authorization_endpoint": "https://login.system.aws-usw02-pr.ice.predix.io",
   "token_endpoint": "https://uaa.system.aws-usw02-pr.ice.predix.io",
   "min_cli_version": null,
   "min_recommended_cli_version": null,
   "api_version": "2.62.0",
   "app_ssh_endpoint": "ssh.system.aws-usw02-pr.ice.predix.io:2222",
   "app_ssh_host_key_fingerprint": "a6:d1:08:0b:b0:cb:9b:5f:c4:ba:44:2a:97:26:19:8a",
   "app_ssh_oauth_client": "ssh-proxy",
   "logging_endpoint": "wss://loggregator.system.aws-usw02-pr.ice.predix.io:443",
   "doppler_logging_endpoint": "wss://doppler.system.aws-usw02-pr.ice.predix.io:443",
   "user": "914a2eda-21ce-4a81-b1d8-0ce568f7f221"
}
cf version 6.21.1+6fd3c9f-2016-08-10
```

What version of the buildpack you are using?
dotnet-core-buildpack 1.0.11

If you were attempting to accomplish a task, what was it you were attempting to do?
Deploy a Dotnet Core app with multiple proyects to GE Predix

What did you expect to happen?
I ran the command `dotnet run -p <path-to-main-project> -b https://github.com/cloudfoundry/dotnet-core-buildpack --server.urls http:0.0.0.0:${PORT}` locally and it worked perfectly. I've followed every step on this guide (https://docs.cloudfoundry.org/buildpacks/dotnet-core/index.html) to deploy a Dotnet Core app to Predix (configure the listen port, create .development file, etc).

What was the actual behavior?
The application fails to start once deployed to Predix.

Please confirm where necessary:
* [x] I have included a log output
* [x] My log includes an error message
* [x] I have included steps for reproduction

Here are the logs obtained from running the command `cf logs <project-name> --recent` after making a `cf push <project-name>` 
[recent.txt](https://github.com/cloudfoundry/dotnet-core-buildpack/files/775057/recent.txt)

**Comment by degrim** [comment]
@Lempiraholio Your application is likely using more memory/disk space than you have assigned to it (this is the usual culprit for -1 exit code).

**Comment by sclevine** [comment]
@Lempiraholio were you able to get this working by including more resources?

**Comment by Lempiraholio** [comment]
Yes, I did! thank you @degrim, the logs weren't as helpful as I wanted. I'll close the issue right now. For the record, dotnet apps need at least 1GB RAM to run.

**Comment by degrim** [comment]
@Lempiraholio How much RAM they need really depends on what you're doing, most apps can run with 512MB using the project.json tooling.  If you're using the MSBuild tooling, I've managed to get an app running with 16MB RAM using this buildpack (it was quite minimal).  But overall, the changes we made in the way that MSBuild apps are run compared to the way we were running apps using the older project.json tooling the MSBuild apps take much less RAM to run.
